packages/debounce/README.md

```diff
@@ -18,8 +18,9 @@ yarn add @solid-primitives/debounce
 ## How to use it
 
 ```ts
-const [fn, clear] = createDebounce(() => console.log('hi'), 250));
+const fn = createDebounce(() => console.log('hi'), 250));
 fn('my-new-value');
```

Comment:
I know that you didn't add the 'my-new-value', but I'm confused by it.

Reply:
If you don't define anything in your callback, you'll get `...args: any` - so that's what fn will accept.

Reply:
not so sure about that one
<img width="352" alt="image" src="https://user-images.githubusercontent.com/24491503/157532682-15f41e98-f57f-46f2-a9de-f31027100a06.png">
`any` by default in not a good option anyway

Reply:
This is the case where there is no argument at all, which should be fine, because the callback receives no argument. If you have an untyped argument, eg.
```ts
createDebounce((untyped) => { ... })
```
then untyped will implicitly be of type `any`, as there is nothing for typescript to infer.

On the other hand, if you type your arguments, they should work as typed.